### PR TITLE
<DropdownLayout/>- align driver and uni driver

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary
This PR includes:
- Aligning the driver to use the option driver 
- Remove all global classname methods (which is not used by our users). This includes removing the         `DATA_OPTION.HOVERED_GLOBAL` and `DATA_OPTION.SELECTED_GLOBAL` data attributes. 
- Rearranging methods order. 

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6184

File: src/DropdownLayout/DropdownLayout.driver.js [hunk 1]
```diff
@@ -42,88 +42,116 @@ const dropdownLayoutDriverFactory = ({ element }) => {
...
+    /** @deprecated */
     isOptionHeightBig: position =>
       doIfOptionExists(
         position,
         () =>
           optionElementAt(position).getAttribute(DATA_OPTION.SIZE) === 'big',
       ),
+
     isShown: () => contentContainer.hasAttribute(DATA_SHOWN),
-    isUp: () =>
```
Comment (TalZa1810) [inline src/DropdownLayout/DropdownLayout.driver.js:108]: moved below `isDown` method.

File: src/DropdownLayout/DropdownLayout.driver.js [hunk 2]
```diff
@@ -42,88 +42,116 @@ const dropdownLayoutDriverFactory = ({ element }) => {
...
+      }),
+
+    mouseLeave: () => ReactTestUtils.Simulate.mouseLeave(element),
+
+    /** @deprecated deprecated prop */
     mouseClickOutside: () =>
       document.body.dispatchEvent(
         new Event('mouseup', { cancelable: true, bubbles: true }),
       ),
-    mouseEnter: () => ReactTestUtils.Simulate.mouseEnter(element),
```
Comment (TalZa1810) [inline src/DropdownLayout/DropdownLayout.driver.js:115]: moved up

File: src/DropdownLayout/DropdownLayout.uni.driver.js [hunk 3]
```diff
@@ -157,22 +155,35 @@ export const dropdownLayoutDriverFactory = base => {
...
+      doIfOptionExists(position, async () => {
+        const optionDriver = await getOptionDriver(position);
+        return optionDriver.mouseLeave();
+      }),
+
     /** @deprecated Use optionDriver*/
-    optionAt: () => optionElementAt.getNative(), // eslint-disable-line no-restricted-properties
-    /** @deprecated This should be a private method since the hook include internal parts ('dropdown-divider-{id}, dropdown-item-{id})') */
+    optionAt: () => {
+      return optionElementAt.getNative(); // eslint-disable-line no-restricted-properties
```
Comment (TalZa1810) [inline src/DropdownLayout/DropdownLayout.uni.driver.js:182]: this method is deprecated. Changed this because of how autodocs parse the eslint comment.

**Review by shlomitc — APPROVED** [review]

**Comment by TalZa1810** [comment]
fails on a visual which is not related. no visual changes.

**Pull request merged**